### PR TITLE
RAC: migrate `<Textarea>` to react-aria components

### PR DESCRIPTION
**Pull request by aromko**

### CODE REVIEW

File: packages/components/src/TextArea/TextArea.test.tsx [hunk 1]
```diff
@@ -70,39 +70,40 @@ test('passes down variant and size', () => {
     />
   );
 
-  const textArea = screen.getByTestId('text-area');
+  const textArea = screen.getByRole('textbox');
   expect(textArea.className).toMatchInlineSnapshot(`"text-lime-500 text-sm"`);
 
   const label = screen.getByText('Label');
+  //console.log(prettyDOM(document));
```
Comment (sarahgm) [inline packages/components/src/TextArea/TextArea.test.tsx:77]: can this be removed? @aromko

File: packages/components/src/TextArea/TextArea.test.tsx [hunk 2]
```diff
@@ -70,39 +70,40 @@ test('passes down variant and size', () => {
...
   expect(label.className).toMatchInlineSnapshot(
     `"text-lime-500 text-sm flex w-[var(--labelWidth)]"`
   );
-
+  /*
```
Comment (sarahgm) [inline packages/components/src/TextArea/TextArea.test.tsx:81]: this also?

File: packages/components/src/TextArea/TextArea.tsx [hunk 3]
```diff
@@ -1,121 +1,71 @@
...
     const classNames = useClassNames({ component: 'TextArea', variant, size });
+
+    const props: RAC.TextFieldProps = {
+      isDisabled: disabled,
+      isReadOnly: readOnly,
+      isInvalid: error,
+      isRequired: required,
+      ...rest,
+    };
+    console.log({ ...props });
```
Comment (sarahgm) [inline packages/components/src/TextArea/TextArea.tsx:62]: remove log @aromko

File: packages/components/src/TextArea/TextArea.test.tsx (hunk 1 above)
Comment (aromko) [reply packages/components/src/TextArea/TextArea.test.tsx:77]: yes

File: packages/components/src/TextArea/TextArea.test.tsx (hunk 2 above)
Comment (aromko) [reply packages/components/src/TextArea/TextArea.test.tsx]: Nope. I fixed it now :-)

**Review by sarahgm — APPROVED** [review]

**Review by sarahgm — APPROVED** [review]

**Pull request merged**